File: Prog2part3.py
def main():
    sv = int(input("Enter starting value: "))
    ev = int(input("Enter ending value: "))

    if sv > ev:
        incr = -1
    else:
        incr = 1

    print()

    for i in range(sv, ev, incr):
        print(i)
    print(ev)

if __name__ == "__main__":
    main()

File: Prog2part3.cpp
#include <iostream>
using namespace std;

int main(){
    int sv, ev, incr;
    
    cout << "Enter starting value: ";
    cin >> sv;

    cout << "Enter ending value: ";
    cin >> ev;

    if (sv > ev){
        incr = -1;
    } else {
        incr = 1;
    }

    cout << endl;

    for (int i = sv; i != ev; i += incr){
        cout << i << endl;
    }

    cout << ev << endl;
}

